scripts/app.js

```diff
@@ -89,8 +93,55 @@ document.addEventListener("keydown", function(event) {
 
         saveContentToCache();
     }
+
```

Comment:
We can isolate the condition on `event.ctrlKey`, to avoid repeating it:

```
if (event.ctrlKey) {
    if (event.code === "Slash") {
        ...
    } else if (...) {
        ...
    }
```

And, given the current structure of the code, a guard clauses pattern can even be used:

```
if (!event.ctrlKey) {
    return;
}

if (event.code === "Slash") {
    ...
} else if (...) {
    ...
}
```